uint8 open_close

uint8 DIR_OPEN=0
uint8 DIR_CLOSE=1
